demo/index.html

```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<title>Typography Snapshot</title>
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
-		<script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
Could also start using [`webcomponents-loader.js`](https://www.polymer-project.org/2.0/docs/polyfills) which will only load the things that are necessary.